### PR TITLE
Add Lax Jax frontend square operator function 

### PR DESCRIPTION
**Pull request by Bambarithm**
This PR addresses the issue https://github.com/unifyai/ivy/issues/2862 which is tracked in https://github.com/unifyai/ivy/issues/1578

Please review.

Thanks

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py [hunk 1]
```diff
@@ -300,3 +300,33 @@ def test_jax_lax_acos(
...
+    ):
+        input_dtype, x = dtype_and_x
+
+        helpers.test_frontend_function(
+            input_dtypes=input_dtype,
+            as_variable_flags=as_variable,
+            with_out=False,
+            num_positional_args=num_positional_args,
+            native_array_flags=native_array,
+            fw="torch",
```
Comment (Ishticode) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py:328]: please add fw argument

**Review by Ishticode — CHANGES_REQUESTED** [review]
Hi @Bambarithm Good PR. Minor change requested. Please request my review when done..

**Comment by Bambarithm** [comment]
Hi @Ishticode thanks for reviewing my PR :smiley:

**Review by Ishticode — CHANGES_REQUESTED** [review]
Hi. So two requests.
Firstly, Could you remove the unsupported part bcz torch does support square function for float16 or bfloat16. Any reason you stated them unsupported that i am unaware of perhaps?

Secondly, Could you try and resolve the conflict if possible. Let me know if you need help for that.
Thanks 😊

**Comment by Bambarithm** [comment]
So I tried removing the unsupported part and part of the test started failing.  I get the following error: 

E           TypeError: can't convert np.ndarray of type bfloat16. The only supported types are: float64, float32, float16, complex64, complex128, int64, int32, int16, int8, uint8, and bool.

3 out of 4 ivy functional backend tests passed except for the torch one which is weird.

Also, I resolved the conflict. Please review it. 

Thanks

**Comment by MrZilinXiao** [comment]
Hi @Bambarithm @Ishticode! I am responsible for checking stale PRs regularly. Since this PR has been inactive for 39 days, I am wondering if this PR is still ongoing. Feel free to reply, or I will close it within 24 hours. In the meanwhile, you could reopen it if you believe the PR is still useful for you.

**Pull request closed without merging**